app/stores/WalletsStore.js

```diff
@@ -218,23 +218,20 @@ export default class WalletsStore extends Store {
     }
   };
 
+  @action _setIsWalletDialogOpen = (isOpen: boolean) => {
+    this.isAddWalletDialogOpen = isOpen;
+  };
+
   goToWalletRoute(walletId: string) {
     const route = this.getWalletRoute(walletId);
     this.actions.router.goToRoute({ route });
   }
 
   _openAddWalletIfNoWallets = () => {
```

Comment:
I would rename this to `_openAddWalletDialogWhenThereAreNoWallets` to make it more explicit